students/adam_wulw/lesson_01_python_basics/car_route.py

```diff
@@ -0,0 +1,6 @@
+km_p_day = int(input('How meny km per day?\n'))
+route = int(input('How far you are going?\n'))
+extra_dey = 0;
+if (route  % km_p_day  > 0):
```

Comment:
It will be always > or ==. Try to change extra_day to to value 1 or 0 in easiest way